src/backend/routes/chat.go

```diff
@@ -23,7 +24,7 @@ import (
 // Initialize a Redis client as a global variable.
 var redisClient *redis.Client
 var ctx = context.Background()
-var chatFetchCmds = make(map[string]*exec.Cmd)
+var chatFetchCmds sync.Map // [string]*exec.Cmd
```

Comment:
Nitpick: remove commented out code. It's in the Git history if we need it.
```suggestion
var chatFetchCmds sync.Map
```